valid-form.php

```diff
@@ -0,0 +1,96 @@
+<?php
+    define("MAX_LENGTH_SUBJECT", 150);
+    define("MAX_LENGTH_LECTURER", 200);
+    define("MIN_LENGTH_DESCRIPTION", 10);
+    define("MAX_NUMBER_CREDITS", 10);
+    define("MIN_NUMBER_CREDITS", 0);
+    define("NUMBER_VALID_INPUTS", 5);
+
+  function phpAlert($msg) {
+    echo '<script type="text/javascript">alert("' . $msg . '")</script>';
```

Comment:
Please avoid using alert to notify the user. More info: https://stackoverflow.com/questions/20359807/why-shouldnt-i-use-alert-in-javascript